### ISSUE TITLE
Question about performance with 500+ options

### ISSUE THREAD

**Issue by sugaaloop** [indrimuska/angular-selector#39]
I'm having performance issues when the option list is 500+ in size, any suggestions of how I could research improvements? Love the tool, by the way.

**Comment by Nishchit14** [comment]
You can use `remote` option and fetch data on demand rather than loading all data first.

**Comment by sugaaloop** [comment]
Thanks for the idea @Nishchit14, however, loading data is not the bottleneck. Creating the select, and I suppose the several ng-repeats, is where the slowdown occurs.

**Comment by indrimuska** [comment]
Yes @sugaaloop, I think so too, but I don't have the time to understand how to pack a fix to improve performances. If you have any idea or you are interested in contribution you're always be welcome! :)

**Comment by andreialecu** [comment]
You can still use the remote option and filter your options within it to only return a few of them that match the user input, instead of the huge list.

**Comment by jeeyyy** [comment]
This is fixed in a high performance variant/fork of the same component:
https://github.com/jkodu/angular-selector-on-steroids

Refer enhanced example page - https://jkodu.github.io/angular-selector-on-steroids/ (allow unsafe scripts).
